automation/run-tests.sh

```diff
@@ -147,7 +147,25 @@ function collect_artifacts {
     "
 }
 
+function write_separator {
```

Comment:
Could you please place some examples on the commit message that shows that this actually does?